src/app/models/manuscript-state.factory.ts

```diff
@@ -30,6 +31,17 @@ export function createTitleState(content: Node): EditorState {
   });
 }
 
+export function createAuthorsState(authorsXml: Element[]): Person[] {
+  return authorsXml.map((author) => ({
+    _id: uuidv4(),
```

Comment:
Not an issue now, but a heads up that I think there can be an id set in the xml. If I recall it's an attribute on the 'ref' element called 'id', so if that is present we should use that as that is further down the line how citations work, you create an 'xref' in the document that links to that 'id'. No sure though if you want to fix this now or further down the line though. I'm not sure if there is a standard for the ID's either, like 'contrib-1', 'contrib-2', etc, which would further make this work fun.

Reply:
Right. Yeah, that's not quite the person object that has it, but it is the id of a reference in the ref list, and it can have multiple people as children. Is that the one you're talking about? Linking of authors happens differently if I'm not mistaken through xref tags rather than ID or contributors. This id is for our internal use only. It's meant to be like a primary key. Devoid of meaning and reasons to change.